admin/class-metabox.php

```diff
@@ -908,6 +586,9 @@ public function enqueue() {
 				'jquery-ui-autocomplete',
 			), WPSEO_VERSION, true );
 
+			wp_enqueue_script( 'wp-seo-wordpressScraper.js', plugins_url( 'js/wp-seo-wordpress-scraper' . WPSEO_CSSJS_SUFFIX . '.js', WPSEO_FILE ), null, WPSEO_VERSION, true );
+			wp_enqueue_script( 'yoast-seo-content-analysis', plugins_url( 'js/dist/yoast-seo-content-analysis.min.js', WPSEO_FILE ), null, WPSEO_VERSION, true );
```

Comment:
This script is now getting enqueued in two places. Might be better if it were only included in one place and only on the pages where it is needed.